test/logic/stake_logic/apy_helper_test.rb

```diff
@@ -9,9 +9,11 @@ class ApyHelperTest < ActiveSupport::TestCase
   end
 
   test "number_of_epochs returns correct number" do
-    num_of_epochs = number_of_epochs(1.day.ago, 2.days.ago)
+    create(:cluster_stat, network: @network, epoch_duration: 159521.0)
```

Comment:
Please set a realistic epoch duration, so we can actually see if the results make sense. Epochs are never shorter than 2 days.